source/_elephpants/2019-06-07-yellow-darkmira.md

```diff
@@ -0,0 +1,8 @@
+---
+categories:
+    - darkmira
+tags:
+    - yellow
```

Comment:
Does it say this on the web page? I couldn't tell.

Reply:
Say what, the color?  I would assume it would match the renderings

Reply:
There's an English version now too: https://php.darkmiratour.rocks/2019/en/elephpant.html

Reply:
👍 I didn't notice the color of the renderings. That's what I get for running Flux!

Reply:
I'm colorblind, so you should double check that it's not green 😁

Reply:
When I turned off Flux it was definitely yellow. 😃